### PR TITLE
added option for no grid

### PR DESCRIPTION
**Pull request by shimwell**
This PR adds the ability to not display a grid as discussed in #12 

There is also a test added to the CI and a new example in the readme

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -63,6 +63,14 @@ You may want to play with the supported options for a customized look:
 
       mypart = cadquery.Workplane("XY").box(2, 2, 2)
 
+Optionally the grid can also be removed by setting gridsize to 0
```
Comment (Peque) [inline README.rst]: End the sentence with a colon (`:`)?

File: sphinxcadquery/sphinxcadquerystatic/main.js [hunk 2]
```diff
@@ -140,11 +140,13 @@ function init() {
 
         var gsize = parseFloat(scene.userData.view.gridsize);
         var gdivs = parseInt(scene.userData.view.griddivisions);
-        var grid = new TranslucentGrid( gsize, gdivs, 0x888888, 0xdddddd, 0.6, );
+        if ( gsize > 0 ){
```
Comment (Peque) [inline sphinxcadquery/sphinxcadquerystatic/main.js]: Add a space between `) {`?

File: tests/example/index.rst [hunk 3]
```diff
@@ -19,4 +19,13 @@ Example 2
...
+   mypart = cadquery.Workplane("XY").box(2, 2, 2)
+
+Example 3
+^^^^^^^^^
+
+.. cadquery::
+   :select: mypart
+   :gridsize: 0
+
+   result = cadquery.Workplane("XY").box(2, 2, 2)
```
Comment (Peque) [inline tests/example/index.rst:30]: You are selecting `mypart` but the variable is `result`. I guess this may throw an error?

**Review by Peque — CHANGES_REQUESTED** [review]

**Comment by Peque** [comment]
@Shimwell Left a couple of comments for your consideration. Also, I would prefer the commit title to be `Add option for no grid` (imperative, with capital letter). :innocent: 

Apparently, I could merge this pull-request right now, but the CI pipeline has not passed (and it is not showing). Do you know which configuration is needed to make pipelines **mandatory** to pass? It can be considered in a new pull-request, and a new issue can be created to deal with that. :blush:

**Comment by shimwell** [comment]
I think the CI pipe line doesn't run from forks by default but this can be enabled https://docs.github.com/en/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository

I think you can make the master branch protected in the github settings 
"After enabling required status checks, all required status checks must pass before collaborators can merge changes into the protected branch."
https://docs.github.com/en/github/administering-a-repository/about-protected-branches#require-status-checks-before-merging

**Pull request merged**